src/app/components/Account/index.tsx

```diff
@@ -56,13 +65,19 @@ export const Account: FC<AccountProps> = ({ account, roseFiatValue }) => {
           value={account.address}
         />
       </dd>
+
+      <dt>{t('account.chain')}</dt>
+      <dd>{t('common.emerald')}</dd>
+
+      <dt>{t('common.balance')}</dt>
+      <dd>{t('common.valueInRose', { value: balance })}</dd>
       {roseFiatValue && (
         <>
           <dt>{t('common.fiatValue')}</dt>
           <dd>
             <StyledBox>
               {t('common.fiatValueInUSD', {
-                value: mockedRuntimeBalance * roseFiatValue,
+                value: parseFloat(balance) * roseFiatValue,
```

Comment:
I think all calculations should be done in BigNumber
(cc @csillag, in case you're adding `fee / gas_limit`)